project.ruleset.xml.example

```diff
@@ -14,6 +14,9 @@
 	<!-- Exclude the Composer Vendor directory. -->
 	<exclude-pattern>/vendor/*</exclude-pattern>
 
+	<!-- Exclude minified Javascript files. -->
+	<exclude-pattern>*.min.js</exclude-pattern>
+
```

Comment:
For this matter, should we also exclude `/node_modules/*` since we're also excluding `vendor`?